components/util/Calendar.js

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+
+const SyledCalendarWrapper = styled.div`
```

Comment:
typo! i think it's supposed to be "StyledCalendarWrapper" instead of "SyledCalendarWrapper" ? :)